README.md

```diff
@@ -116,7 +129,11 @@ body, that says "hello".
 
 Your Answer:
 ```js
-// your code here
+
+    $("button#greeting").on("click", function(){
```

Comment:
While this will work, some linters will give you a warning about combining tag AND id selectors.  The id is (should be) unique.